plugin/trino-hive/src/test/java/io/trino/plugin/hive/util/TestHiveBucketing.java

```diff
@@ -365,7 +365,7 @@ public static int getHiveBucketHashCode(BucketingVersion bucketingVersion, List<
         throw new IllegalArgumentException("Unsupported bucketing version: " + bucketingVersion);
     }
 
-    private static Object toNativeContainerValue(Type type, Object hiveValue)
+    public static Object toNativeContainerValue(Type type, Object hiveValue)
```

Comment:
This should move to `HiveTestUtils` now that we use it from outside this class

Reply:
Sounds good, thanks